src/components/Typography/Typography.tsx

```diff
@@ -8,7 +8,7 @@ interface Props extends TextProps {
 
 export const Typography: React.FC<Props> = ({ type, children, ...props }) => {
   return (
-    <Text {...props} style={[styles[type], props.style]}>
+    <Text {...props} style={[styles[type], styles.fontFamily, props.style]}>
```

Comment:
In my opinion, is better to use a name that is more expandable, something like `defaultText`.